### DISCUSSION TITLE
SlickGrid Error when opening Forms with multiple Tabs - Only occurs on deployed App and not Local Host

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6641] [Help]
I recently upgraded my Serenity codebase to .Net6

Now, all my forms with that have multiple Tabs (e.g. with a Template.html file)  are getting this error when I open the form:

![image](https://user-images.githubusercontent.com/57414714/211110369-5097e85b-e8e9-4e90-abc8-5d87a2d574a4.png)

This error only occurs on deployed Web App, and not on Local Host.

How do I solve this issue?

This is the same as this older issue
https://github.com/serenity-is/Serenity/issues/4887

**Comment by cletrop** [comment]
Por si a alguien le sirve, yo lo solucione con las opciones de compilación del archivo, te ubicas en el archivo, click derecho, Propiedades, y te fijas que `Build Action` este en `None` y que `Copy to output directory` este en `Copy always`, este ultimo ayuda a que aun cuando publiques la aplicación también ese `XYZ.Template.html` se vaya en la compilación
 
![image](https://github.com/serenity-is/Serenity/assets/32921735/3c9d4adc-84ef-42e0-9da0-e644a47c9a29)
